.github/workflows/generate-code.yml

```diff
@@ -64,6 +64,21 @@ jobs:
 
           echo "DIFF_IS_EMPTY=$([[ -z "$diff_excluding_submodule" ]] && echo 'true' || echo 'false')" >> $GITHUB_ENV
           echo "CURRENT_DATETIME=$(date +'%Y%m%d%H%M%S')" >> $GITHUB_ENV
+
+      # Save full diff to file and upload artifact
```

Comment:
I recognize this as a workaround, but do you encounter a diff when executing the steps written in the workflow locally?
https://github.com/line/line-bot-sdk-php/blob/e6a184966872b8dbceb02eb7c19624c066d3c2f7/.github/workflows/generate-code.yml#L33-L57


If so, this might confuse external contributors too like you, and we should consider measures to ensure stricter version control. For example, https://github.com/line/line-bot-sdk-php/blob/master/CONTRIBUTING.md should be updated.

.... However, since CI provides stable results, could it be that your local version is misaligned...?
@eucyt might help you when you're confused with something.

Reply:
It seems the PHP version I was running locally was outdated. The issue is now resolved, but I plan to keep this change for future troubleshooting.